docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md

```diff
@@ -0,0 +1,76 @@
+| Image Pull Secret                                                                                                                                                                                                                                              | If your application will run private image(s), specify an ImagePullSecret that will be used to authenticate with your registry. If the specified ImagePullSecret is not correct or missing, the image pulling and application deployment operations will fail. |
+| Volumes                                                                                                                                                                                                                                                        | If Pod Controller is set to StatefulSet, add one or more volumes by defining their name, storage class, and capacity.                                                                                                                                          |
+| Container                                                                                                                                                                                                                                                      | Specify the container name, docker image and tag, CPU and memory resources, environment variables, and volume mounts for the first container (Container0).<br>Use Advanced Settings to define the following parameters:<br><ul><li><strong>Command</strong> = entrypoint array in JSON or YAML format. If omitted, the docker image's ENTRYPOINT is used. Variable references $(VAR_NAME) are expanded using the container's environment.</li><li><strong>Args</strong> = arguments to the entrypoint in JSON or YAML format. If omitted, the docker image's CMD is used. Variable references are expanded using the container's environment.</li><li><strong>Image Pull Policy</strong> =  select from Always, Never, or IfNotPresent. If latest tag is specified, this setting defaults to Always; otherwise, it defaults to IfNotPresent</li><li><strong>Liveness Probe</strong> = periodic probe of container service readiness. If the probe fails, the container will be removed from service endpoints.</li><li><strong>Readiness Probe</strong> = periodic probe of container service readiness. If the probe fails, the container will be removed from service endpoints.</br></li>                                                               |
+
+**Note:** You can add containers, environment variables, and volume mounts by clicking their **\+ Add** links in the **Container** section.
+
+5. At the top right of the wizard, click **\> Next**. 
+6. Select whether to expose your application using a [Layer 4 load balancer](<#exposing-via-a-load-balancer>) or the[ public network](<#exposing-via-the-public-network>), and then complete the remaining fields based on your selection.
```

Comment:
Text changed. Let's touch base about linked content.

Reply:
The linked content is the content you'll see when clicking the link "public network interfaces of a pod replica". Same change needs to be made on the linked content to avoid simply saying "public network".

You can check the current linked content here https://github.com/mileweb/docs-content-ecp/blob/marc-dev/docs/portal/applications/adding-and-deploying-applications-using-a-wizard.md#exposing-via-the-public-network

Reply:
Fixed.

Reply:
The link behind "public network interfaces of a pod replica" is broken. Please check.
Please change the table heading from "Fields for Exposing Application via the Public Network" to "Fields for Exposing Application via the Public Network Interfaces of a Pod Replica?" Please do include "Replica" at the end.